YAMP.Core/Help/Documentation.cs

```diff
@@ -307,7 +307,9 @@ static String GetLocalized(String key)
         {
             var value = default(String);
             var dictionary = Localization.Current ?? Localization.Default;
-            dictionary.TryGetValue(key, out value);
+            //Fallback to show something while key/descriptions are being replaced
+            if (!dictionary.TryGetValue(key, out value))
+                value = key;
```

Comment:
I also thought about this fallback, but I decided against it for (among others) testing purposes. Returning `null` allows me to automatically spot if a mapping is still missing.